gn2/wqflask/templates/genewiki.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+{% block title %}GeneWiki Entry for {{ symbol }}{% endblock %}
+{% block css %}<style></style>{% endblock %}
+{% block content %}
+    <div class = "container">
+        <h1 class = "page-header">GeneWiki Entries</h1>
+        <p>
+            GeneWiki enables you to enrich the annotation of genes and transcripts. Please submit or edit a GeneWiki note (500 characters max) related to a gene, its transcripts, or proteins. When possible include PubMed identifiers or web resource links (URL addresses). Please ensure that the additions will have widespread use. For additional information, check the GeneWiki <a href="https://gn1.genenetwork.org/GeneWikihelp.html" target="_blank">help document</a>.
+        </p>
+        <h3>GeneWiki For {{ symbol }}:</h3>
+        <h5>
+            <strong>GeneNetwork:</strong>
+        </h5>
+        {% if wiki %}
+            <ol>
+                {% for entry in wiki %}
+                    <li>
+                        {{ entry.get("comment") }}
+                        {% if entry.get("web_url") %}
+                            <sup> <small> <a href = "{{ entry.web_url }}" target = "_blank"> <span class = "glyphicon glyphicon-globe" aria-hidden = "true"></span>web</a></small></sup>
+                        {% endif %}
```

Comment:
can we add the history and edit buttons here?

Reply:
Let's add this in another PR.  Let's introduce said buttons when we weave in the functionality---easier on the user.